parm/config/config.fcst

```diff
@@ -155,26 +155,17 @@ if [[ "$OUTPUT_FILE" == "netcdf" ]]; then
     export  ichunk2d=0; export jchunk2d=0
     export  ichunk3d=0; export jchunk3d=0;  export kchunk3d=0
     RESTILE=`echo $CASE |cut -c 2-`
-    if [[ "$machine" == "WCOSS_DELL_P3" ]] || [[ "$machine" == "WCOSS2" ]]; then
+    if [ $RESTILE -ge 768 ]; then
+        export OUTPUT_FILETYPES=" 'netcdf_parallel' 'netcdf_parallel' "
```

Comment:
I don't think this is just for C768 or greater.  Looking at the left, this is being set for HERA and ORION for all resolutions. except less than (or equal) to C192.

Reply:
Have adjusted the OUTPUT_FILETYPES settings again, please review new changes in config.fcst.